doc/guides/backporting-to-release-lines.md

```diff
@@ -32,8 +32,8 @@ branch. In order to submit a backport pull request to another branch, simply
 replace that with the staging branch for the targeted release line.
 
 1. Checkout the staging branch for the targeted release line
-2. Make sure that the local staging branch is up to date with the remote
-3. Create a new branch off of the staging branch
+1. Make sure that the local staging branch is up to date with the remote
+1. Create a new branch off of the staging branch
```

Comment:
I prefer the numbering of the steps before this change (i.e. When it didn't reset to 1).

Reply:
Ok, I checked and most files use `1. 2. 3...` even though there is at least one file that uses `1. 1. 1...`. I personally prefer the latter as it allows to reorder things easily / removing a single entry without creating any churn.

Reply:
My point is that this change has renumbered the steps so that instead of steps 1-10 it now reads steps 1-3, 1, 1-6.

Reply:
You are right. I did not check that anymore. I just removed the commit as our style currently does not really rely on it. I guess it might be introduced as a general thing or it should not be used further at all.